src/Models/Question.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Models;
+
+use Engelsystem\Models\User\User;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property integer $id
+ * @property integer $enquirer_id
+ * @property string $question
+ * @property string $answer
+ * @property integer $answerer_id
+ * @property-read \Engelsystem\Models\User\User $answerer
+ * @property-read \Engelsystem\Models\User\User $enquirer
+ * @method static Builder|\Engelsystem\Models\Question newModelQuery()
```

Comment:
These are (now) in the parent model so no need for these methods ;)

Reply:
✓